src/cluster.c

```diff
@@ -1857,10 +1857,10 @@ void ShardGroupGet(RedisRaftCtx *rr, RedisModuleCtx *ctx)
         node_count++;
         RedisModule_ReplyWithArray(ctx, 2);
 
-        char node_id[RAFT_SHARDGROUP_NODEID_LEN + 1];
+        char node_id[RAFT_SHARDGROUP_NODEID_LEN * 2];
         raft_node_id_t id = raft_node_get_id(raft_node);
 
-        snprintf(node_id, sizeof(node_id), "%s%08x", rr->log->dbid, id);
+        snprintf(node_id, sizeof(node_id), "%s%08x", rr->meta.dbid, id);
```

Comment:
elsewhere we use

```
    char node_id[RAFT_SHARDGROUP_NODEID_LEN + 1];
    raftNodeToString(node_id, rr->meta.dbid, raft_node);
```

which basically translates to

```
    snprintf(output, RAFT_SHARDGROUP_NODEID_LEN + 1, "%.32s%08x", dbid, raft_id);
```

why not use the same code to avoid the complaining issue?

Reply:
ok, used `raftNodeToString()`.